app/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
 
     <application
+        android:networkSecurityConfig="@xml/network_security_config"
```

Comment:
What about security of app with this file ?
Users can get clear traffic ?
If we are in this case delete this file

Reply:
Resolved by commenting respective line and adding explanation comment.